validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorConfig.java

```diff
@@ -679,11 +692,13 @@ private void validateExternalSignerURLScheme() {
         return;
       }
 
-      if (validatorExternalSignerKeystore != null || validatorExternalSignerTruststore != null) {
+      if (validatorExternalSignerKeystore != null
+          || validatorExternalSignerTruststore != null
+          || unsanitizedValidatorExternalSignerUrl.getUserInfo() != null) {
```

Comment:
Not a fan of having a unsanitized copy here... can't we simply expose the sanitized url and Optional userinfo (BasicAuth) as a separate config?

Reply:
Yeah, I agree. I changed it to expose the `userInfo` as an optional.